readme.md

```diff
@@ -251,6 +251,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
   - [Coupling the Flexibility of OVN with the Efficiency of IOVisor](http://openvswitch.org/support/ovscon2016/7/1245-bertrone.pdf)
 
 - [Katran](https://code.fb.com/open-source/open-sourcing-katran-a-scalable-network-load-balancer/) - A layer 4 load-balancer based on XDP, open-sourced by Facebook.
+- [Kernel-collector](https://github.com/netdata/kernel-collector) - Linux Kernel eBPF Collectors
```

Comment:
I don't believe the two links are required, one is probably enough. Let's remove one?

Also whichever link we keep, I'd rater have it at the location (in the list) you used for the Netdata docs. It feels more natural to see it amongst other tools for monitoring rather than XDP-based networking applications.

Reply:
I will keep the one pointing to the docs. Thanks for the review